Mobile Test Runner for Jama/Mobile Test Runner for Jama/TestStepViewController.swift

```diff
@@ -71,6 +86,93 @@ class TestStepViewController: UIViewController, UITextViewDelegate {
+        notesTextViewHeightConstraint.constant = 0
+        //Set the other buttons images to right chevrons
+        expectedResultButton.setImage(UIImage.init(named: rightArrowStr), for: .normal)
+        notesButton.setImage(UIImage.init(named: rightArrowStr), for: .normal)
+        //scroll the text view to the top
+        actionTextView.scrollRectToVisible(CGRect.zero, animated: false)
+    }
+    
+    func expandExpectedResultsTextView() {
+        //If the text view is showing collapse, otherwise expand in remaining space and set the button image to down arrow
+        if expectedResultsTextViewHeightConstraint.constant != 0 {
+            expectedResultsTextViewHeightConstraint.constant = 0
+            expectedResultButton.setImage(UIImage.init(named: rightArrowStr), for: .normal)
+        } else {
+            //Make the text view as large as it can be while still fitting in its superview.
+            expectedResultsTextViewHeightConstraint.constant = (expectedResultsTextView.superview?.frame.height)! * textViewHeightRatio - titleAndDividerHeight - totalSpaceBetweenButtons
+            expectedResultButton.setImage(UIImage.init(named: downArrowStr), for: .normal)
+        }
+        //Set the other text views heights to 0
+        actionTextViewHeightContraint.constant = 0
+        notesTextViewHeightConstraint.constant = 0
+        //Set the other buttons images to right chevrons
+        actionButton.setImage(UIImage.init(named: rightArrowStr), for: .normal)
+        notesButton.setImage(UIImage.init(named: rightArrowStr), for: .normal)
+        //scroll the text view to the top
+        expectedResultsTextView.scrollRectToVisible(CGRect.zero, animated: false)
+    }
+    
+    func expandNotesTextView() {
+        //If the text view is showing collapse, otherwise expand in remaining space and set the button image to down arrow
+        if notesTextViewHeightConstraint.constant != 0 {
+            notesTextViewHeightConstraint.constant = 0
+            notesButton.setImage(UIImage.init(named: rightArrowStr), for: .normal)
+        } else {
+            //Make the text view as large as it can be while still fitting in its superview.
+            notesTextViewHeightConstraint.constant = (notesTextView.superview?.frame.height)! * textViewHeightRatio - titleAndDividerHeight - totalSpaceBetweenButtons
+            notesButton.setImage(UIImage.init(named: downArrowStr), for: .normal)
+        }
+        //Set the other text views heights to 0
+        actionTextViewHeightContraint.constant = 0
+        expectedResultsTextViewHeightConstraint.constant = 0
+        //Set the other buttons images to right chevrons
+        actionButton.setImage(UIImage.init(named: rightArrowStr), for: .normal)
+        expectedResultButton.setImage(UIImage.init(named: rightArrowStr), for: .normal)
+        //scroll the text view to the top
+        notesTextView.scrollRectToVisible(CGRect.zero, animated: false)
+    }
```

Comment:
Perhaps even the section that sets the other two textviews and their images to the collapsed display could be parameterized? I'm seeing the same pattern used three times and wonder if there's a way to condense it.